src/bindgen/parser.rs

```diff
@@ -181,6 +181,8 @@ impl<'a> Parser<'a> {
             }
         }
 
+        self.out.source_files = self.cache_src.keys().map(|k| k.to_owned()).collect();
```

Comment:
So this order is basically random, I'd rather have a well defined order by pushing the path in the `Entry::Vacant` case in `parse_mod`. Otherwise let's have some defined order in the depfile generation?

Reply:
I pushed a commit which sorts the output before writing. This way all source files are sorted, and the config file (if any) is the last entry. This should make writing tests much easier, since it allows us to just compare an output file with an expected file.